assets/js/modules/analytics/components/dashboard/DashboardAllTrafficWidgetV2/index.js

```diff
@@ -0,0 +1,75 @@
+/**
+ * DashboardAllTrafficWidget component
+ *
+ * Site Kit by Google, Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * WordPress dependencies
+ */
+import { _x } from '@wordpress/i18n';
+
+/**
+ * Internal dependencies
+ */
+import Data from 'googlesitekit-data';
+import Widgets from 'googlesitekit-widgets';
+import { STORE_NAME as MODULES_ANALYTICS, FORM_ALL_TRAFFIC_WIDGET } from '../../../datastore/constants';
+import { STORE_NAME as CORE_FORMS } from '../../../../../googlesitekit/datastore/forms/constants';
+import whenActive from '../../../../../util/when-active';
+import TotalUserCount from './TotalUserCount';
+import UserCountGraph from './UserCountGraph';
+import DimensionTabs from './DimensionTabs';
+import UserDimensionsPieChart from './UserDimensionsPieChart';
+import SourceLink from '../../../../../components/SourceLink';
+import { Grid, Row, Cell } from '../../../../../material-components/layout';
+const { Widget } = Widgets.components;
+const { useSelect } = Data;
+
+function DashboardAllTrafficWidget() {
```

Comment:
@aaemnnosttv @tofumatt Should we maybe place this component in `assets/js/modules/analytics/components/dashboard/DashboardAllTrafficWidgetV2.js` instead? It results in the same import path, but my initial impression is that having it in an `index.js` file here is more confusing. If we have it in its own file with the same name of the folder, then it's clear to me that that folder has components related to that widget. Right now though, it's not apparent that the `index.js` file also contains its own component.

Reply:
This is how Node imports tend to work, re: index files. I suppose this one is down to preferences/expectations, but my _usual_ habit is actually to do what @kostyalmm did with `index.js`. All of the components, including the "main" component are in this folder. Otherwise there's components in a folder and a filename. A lot of editors will organise them separately (Atom and VSCode group folders and files separately by default) which makes them harder to see together.

The current approach is what I would expect in a JS project.